src/test/resources/spec/concordion/annotation/Resources.html

```diff
@@ -0,0 +1,218 @@
+<html xmlns:concordion="http://www.concordion.org/2007/concordion">
+<head>
+    <link href="../../../../concordion.css" rel="stylesheet" type="text/css" />
+    <title>Resources</title>
+    <style>
+	    .annotationSummary td, .annotationSummary th {
+	      padding: 8px;
+	      line-height: 135%;
+	    }
+	    .annotationName {
+	      font-family: Courier New, Courier;
+	  	  font-size: 10pt;
+	  	  vertical-align: top;
+	    }
+    </style>
+</head>
+<body>
+
+    <h1>Resources Annotation</h1>
+
+    <p>
+        The Resources annotation can be used to apply a new theme to your specifications, tweak the existing styling, or add 
+        new css, javascript, images, or other resources.
+    </p>
+    <p>
+		The Resources annotation can be applied to all classes in the class hierarchy - each class will be inspected in turn (parent first) 
```

Comment:
For clarity, how about:

``The Resources annotation is applied to the fixture class corresponding to the specification. It can be applied to all classes in the fixture's class hierarchy ...```